meta/inspiredminds/contao-address-verification/de.yml

```diff
@@ -0,0 +1,4 @@
+de:
+    title: Address Verification
```

Comment:
Do you want to have the title in English here on purpose?

Reply:
Yes, as I think it makes no sense to translate names 😉. For example, my OS is not called Microsoft Fenster in German :Þ

Reply:
If you treat this as a proper name, then yes. Otherwise, this would translate to "Adressverifizierung" or something like that ;)

Reply:
Yes, I treat it as a name. Like the name of any Software.